src/preview/viewer.ts

```diff
@@ -28,9 +28,9 @@ export {
 export { serializer } from './viewer/pdfviewerpanel'
 export { hook } from './viewer/pdfviewerhook'
 
-lw.watcher.pdf.onChange(pdfPath => {
-    if (lw.compile.compiledPDFWriting === 0 || path.relative(lw.compile.compiledPDFPath, pdfPath) !== '') {
-        refresh(pdfPath)
```

Comment:
Different. The path check cannot be removed.

Reply:
It's now using `Uri#fsPath` which should likely work as expected.